tests/lib/managers/webhooks-test.js

```diff
@@ -153,6 +153,26 @@ describe('Webhooks', function() {
 
 	});
 
+	describe('setSignatureKeys()', function() {
+		const PRIMARY_SIGNATURE_KEY = 'SamplePrimaryKey';
+		const SECONDARY_SIGNATURE_KEY = 'SampleSecondaryKey';
+
+		it('should validate the primary signature key is set', function() {
+			Webhooks.setSignatureKeys(PRIMARY_SIGNATURE_KEY, SECONDARY_SIGNATURE_KEY);
+
+			assert.equal(PRIMARY_SIGNATURE_KEY, Webhooks.primarySignatureKey);
```

Comment:
The order of the `assert.equal()` parameters in these tests is reversed.  Chai takes the actual value first, and the expected value second (see http://chaijs.com/api/assert/#method_equal).  This is important for error messaging when the tests fail.